autowiring/CoreContext.h

```diff
@@ -290,7 +291,7 @@ class CoreContext:
   // Enables a boltable class
   template<typename T, typename... Sigils>
   void EnableInternal(T*, Boltable<Sigils...>*) {
-    [](...){}((AutoRequireMicroBolt<T, Sigils>(),false)...);
+    autowiring::noop((AutoRequireMicroBolt<T, Sigils>(),false)...);
```

Comment:
I tried removing the `,false` and it didn't compile. You can't pass `void` as an argument to a function.